### PR TITLE
[FSDE-16083] Migrate Activity component to vue documentation

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

File: src/documentation/components/Navigation.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
         <div class="chi-collection__title">Components</div>
```
Comment (evansito) [inline src/documentation/components/Navigation.vue:8]: the above avatar image shouldn't be in this PR

File: src/documentation/pages/components/activity/examples/_with-avatar.vue [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                div
+                  | 11:37 PM
+              .chi-activity__item-avatar
+                .chi-avatar
+                  img(src='../../../../assets/images/avatar.jpg', alt="avatar")
+              .chi-activity__end
+                .chi-activity__end-header
+                  |  Password change for user trent.anderson
+                .chi-activity__end-footer
+                  | Admin
```
Comment (evansito) [inline src/documentation/pages/components/activity/examples/_with-avatar.vue:44]: could we use a v-for loop here?

File: src/documentation/pages/components/activity/examples/index.vue [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<template lang="pug">
+  div
+    h3 Examples
+    p.-text
+      | To render an avatar, use the class <code>chi-activity</code>.
```
Comment (evansito) [inline src/documentation/pages/components/activity/examples/index.vue:5]: To render an activity, not avatar

**Review by evansito — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://2b2e87d30919e36b82c15e37eb0a6eda70bcc854-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-343-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/4/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/5/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://95dea065fe0b845cb1755970185757a7b7683e2e-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-343-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by josecarlosgonzalezv — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/6/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ca72ef42c83f218f95c1528cef582b3be9e8a3c7-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-343-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/7/. :x:

**Review by evansito — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-343/8/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://d75f049ec1b9d04fa0a3e58d8467834a09b7c77a-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-343-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Review by jllr — APPROVED** [review]

**Pull request merged**